lk2nd/device/dts/msm8953/msm8953-asus-zenfone3.dts

```diff
@@ -23,4 +23,32 @@
 			gpios = <&tlmm 13 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
 		};
 	};
+
+	panel {
+		compatible = "asus,zenfone3-panel", "lk2nd,panel";
+
+		qcom,mdss_dsi_ctc5p5_ili7807b_1080p_video {
+			compatible = "asus,ctc5p5_ili7807b";
+		};
+
+		qcom,mdss_dsi_tm5p5_r63350_1080p_video {
+			compatible = "asus,tm5p5_r63350";
+		};
+
+		qcom,mdss_dsi_tm5p2_r63350_1080p_video {
+			compatible = "asus,tm5p2_r63350";
+		};
+
+		qcom,mdss_dsi_boe5p2_ili7807b_1080p_video {
+			compatible = "asus,boe5p2_ili7807b";
+		};
+
+		qcom,mdss_dsi_txd5p5_nt35596_1080p_video {
+			compatible = "asus,txd5p5_nt35596";
+		};
+
+		qcom,mdss_dsi_lce5p5_otm1901a_1080p_video {
+			compatible = "asus,lce5p5_otm1901a";
+		};
```

Comment:
Something tells me that "5p5" and "5p2" are display panel sizes, it seems like this device is 5.2?

```suggestion
		qcom,mdss_dsi_tm5p2_r63350_1080p_video {
			compatible = "asus,tm5p2_r63350";
		};

		qcom,mdss_dsi_boe5p2_ili7807b_1080p_video {
			compatible = "asus,boe5p2_ili7807b";
		};
```

Reply:
ah great those are two different phones...